crates/inspection-service/src/inspection_service.rs

```diff
@@ -20,13 +20,13 @@ use std::{
     thread,
 };
 
-// TODO: we need to write tests for this service!
-
 // Useful string constants
 const CONTENT_TYPE_JSON: &str = "application/json";
 const CONTENT_TYPE_TEXT: &str = "text/plain";
-const DISABLED_ENDPOINT_MESSAGE: &str =
-    "This endpoint is disabled! Enable it in the InspectionServiceConfig.";
+const CONFIGURATION_DISABLED_MESSAGE: &str =
+    "This endpoint is disabled! Enable it in node.yaml inspection_service.expose_configuration: true";
```

Comment:
nit: not all operators call their config files `node.yaml`. I think the convention is `public_full_node.yaml`, `validator.yaml` and `validator_full_node.yaml`.

But, my feeling would be to avoid the issue entirely and leave it unnamed, e.g., `Enable it in your node config via inspection_service.expose_configuration: true`